plugins/in_tail/tail_file.c

```diff
@@ -979,7 +979,12 @@ int flb_tail_file_to_event(struct flb_tail_file *file)
         return -1;
     }
 
-    if (flb_tail_file_name_cmp(name, file) != 0) {
+    /*
+     * flb_tail_target_file_name_cmp is a deeper compare than
+     * flb_tail_file_name_cmp. If applicable, it compares to the underying
```

Comment:
Not related to the logic, but your comment has a typo:

`underying -> underlying`

Reply:
I will fix!